package.json

```
@@ -62,12 +63,21 @@
           "markdownDescription": "Show successfully parsed tokens by the sway-lsp server as warnings. If set to false, sway-lsp will revert to only showing warnings and errors reported by the compiler.",
           "default": false,
           "type": "boolean"
+        },
+        "sway-lsp.trace.fuel-core.logfile": {
+          "description": "Set the file path for the Fuel core log output. Disabled if set to empty string.",
+          "type": "string",
+          "default": "/tmp/fuel-core.log"
```

Just to avoid collisions a bit more
```suggestion
          "default": "/tmp/fuel-core_sway-vscode-plugin.log"
```